CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 - Added `asset_vault::peek_balance` ([#1745](https://github.com/0xMiden/miden-base/pull/1745)).
 - Added `get_auth_scheme` method to `AccountComponentInterface` and `AccountInterface` for better authentication scheme extraction ([#1759](https://github.com/0xMiden/miden-base/pull/1759)).
 - Added `AddressInterface` type to represent the interface of the account to which an `Address` points ([#1761](https://github.com/0xMiden/miden-base/pull/#1761)).
+- Document `miden` library procedures and the context from which they can be called ([#1799](https://github.com/0xMiden/miden-base/pull/#1799)).
```

Comment:
Since we had a release of the `v0.11.0`, should this line go to the `## 0.12.0 (TBD)` section?

Reply:
I think since this is getting merged into `main`, the current placement is fine.